test/functional/mining_basic.py

```diff
@@ -73,6 +76,42 @@ def mine_chain(self):
         self.restart_node(0)
         self.connect_nodes(0, 1)
 
+    def test_blockmintxfee_parameter(self):
+        self.log.info("Test -blockmintxfee setting")
+        self.restart_node(0, extra_args=['-minrelaytxfee=0'])
+        node = self.nodes[0]
+
+        # test default (no parameter), disabled (=0) and a bunch of arbitrary minimum tx fee rates [sat/kvB]
```

Comment:
Are you sure `=0` means disabled? AFAICT it is still enforced, but you can only hit it using `prioritisetransaction` to make the fee negative. "Disabled" suggests to me that the block assembler would include packages at any feerate.

Reply:
You're right. I think the "disabled" string originated from sloppily copy/pasting the test loop structure from mempool_dust.py. Replaced by "zero" now.